Reference/Management/Services/MediaService/Index.md

```diff
@@ -1,22 +1,17 @@
 ---
-versionFrom: 6.0.0
-needsV8Update: "true"
+versionFrom: 8.0.0
 ---
 
 # MediaService
 
-:::note
-Applies to Umbraco 6.0.0+
-:::
-
-The MediaService acts as a "gateway" to Umbraco data for operations which are related to Media.
+The MediaService acts as a "gateway" to Umbraco data for operations which are related to media.
 
-[Browse the API documentation for MediaService](https://our.umbraco.com/apidocs/v7/csharp/api/Umbraco.Core.Services.MediaService.html).
+[Browse the API documentation for IDomainService interface](https://our.umbraco.com/apidocs/v8/csharp/api/Umbraco.Core.Services.IMediaService.html).
```

Comment:
Think this should be IMediaService?

Reply:
@Jeavon thanks 😄 I was also writing documentation for the domain, so seems like I missed changing this one.